tattoo.ts

```diff
@@ -420,10 +430,29 @@ async function _main(elements: ElementRepo[]) {
     "repos/mocha",
     "repos/marked"
   ]);
-  const testProgress =
-      standardProgressBar("Testing...", elements.length);
   const testPromises: Array<Promise<TestResult>> = [];
-  for (const element of elements) {
+
+  let elementsToTest: ElementRepo[];
+
+  if (typeof opts["test-repo"] === "string") {
```

Comment:
This can be moved into the config for `test-repo` to command-line-args, like this version in tedium: https://github.com/PolymerLabs/tedium/blob/4ce1cb6bafcda9f32fd5e7353f401fcaa397b23b/tedium.ts#L68-L85